frontend/src/scenes/session-recordings/player/list/listLogic.ts

```diff
@@ -59,6 +62,15 @@ export const listLogic = kea<listLogicType>([
                 setRenderedRows: (_, { renderMeta }) => renderMeta,
             },
         ],
+        expandedRows: [
+            new Set<number>(),
+            {
+                expandRow: (state, { rowIndex }) =>
+                    rowIndex === undefined ? state : new Set([...Array.from(state), rowIndex]),
+                collapseRow: (state, { rowIndex }) =>
+                    rowIndex === undefined ? state : new Set(Array.from(state).filter((index) => index !== rowIndex)),
+            },
+        ],
```

Comment:
Bubbling up expanded rows to parent component so that we can eventually unblock work for navigating to matching events from url.